### PR TITLE
add extra documentation on requirrements for project copy

### PR DESCRIPTION
**Pull request by Tooyosi**
Add requirements to project copy doc

### CODE REVIEW

File: panoptes_client/project.py [hunk 1]
```diff
@@ -205,6 +205,9 @@ def copy(self, new_subject_set_name=None):
         used to name a new SubjectSet for the copied project.
         This is useful for having an upload target straight after cloning.
 
+        Requirements: the source project must not be live and project.configuration
+        must have a template key (typically, set to true).
```
Comment (lcjohnso) [inline panoptes_client/project.py:209]:
```suggestion
        Requirements: the source project must not be live (in development mode) and 
        project.configuration must have a `template` key set to "true".
```

Comment (lcjohnso) [reply panoptes_client/project.py:209]: Minor edit suggestion

**Review by lcjohnso — APPROVED** [review]
LGTM, but I suggested a minor edit for clarity & formatting

**Comment by PmasonFF** [comment]
These new functionalities ( eg project.copy) are  nice but you all are aware that many people are having major issues getting the panoptes client or the CLI to function for long enough to upload even as few as hundreds of subjects, or to maintain  connection through more than a few hundred requests from the  API ?

It is NOT just the updates to Python 3.12 and the SSL security changes after 3.10.   Issue 220 is a  continual pain now with "stale object" errors hobbling even older virtual environments using set-up that used to work....

I am tell my teams to just keep sending the uploads until they succeed, preferably in off peak times but often there has been no success after many tries (six or more!)

Peter

**Pull request merged**